javascript/components/MapView.js

```diff
@@ -221,7 +238,9 @@ class MapView extends React.Component {
     logoEnabled: true,
     zoomLevel: 16,
     userTrackingMode: MapboxGL.UserTrackingModes.None,
+    //userLocationVerticalAlignment: MapboxGL.UserLocationVerticalAlignment.Center,
```

Comment:
I'm assuming this is from your fork would you mind removing? and squashing down the commits